repository-data/webfiles/src/main/resources/site/freemarker/publication/publication.ftl

```diff
@@ -12,9 +15,26 @@
     <div id="taxonomy">
         <h4>Taxonomy:</h4>
         <#if document.keys?has_content>
-            <#list document.keys as key>
-                ${key?html}
-            </#list>
+           <#list document.keys as key>
+               <div>
+                   <#if taxonomy.getCategoryByKey(key).getParent()??>
```

Comment:
it might be nicer to expose simple function in Component like `List<TaxonomyCategory> getCategoriesChainByKey(key)` (not sure about the TaxonomyCategory class name)